s3-import-commons/src/test/java/no/unit/nva/publication/s3imports/DeleteImportCandidatesEventEmitterTest.java

```diff
@@ -46,8 +45,7 @@ public class DeleteImportCandidatesEventEmitterTest {
 
     public static final Context CONTEXT = Mockito.mock(Context.class);
     public static final long SOME_FILE_SIZE = 100L;
-    public static final InputStream EVENT = IoUtils.inputStreamFromResources(
```

Comment:
Naughty. Cannot be consumed more than once, which means `@RepeatedTest` doesn't work, which I want it to do when I am looking to see if I can provoke an error.